pages/desktop/frontend/search.py

```diff
@@ -123,10 +133,13 @@ def name(self):
                 return self.find_element(*self._search_item_name_locator).text
 
             def link(self):
+                self.wait.until(
+                    EC.element_to_be_clickable(self._search_item_name_locator)
+                )
                 self.find_element(*self._search_item_name_locator).click()
                 from pages.desktop.frontend.details import Detail
 
-                detail_page = Detail(self.selenium, self.page.base_url)
+                detail_page = Detail(self.selenium, self.page.page.base_url)
```

Comment:
Is this correct? 2 ```page``` attributes?

Reply:
Yes, beacuse this class gets as page attribute a SearchResultList(Region) which which gets as page attribute a  Search(Base). And we want to acces the base_url from the Search class. I tested this when I made a validating function for sort by 'Last Update'. It did not work with only one page attribute, but worked with two instead.